cli/create.go

```diff
@@ -259,6 +259,16 @@ func createContainer(ociSpec oci.CompatOCISpec, containerID, bundlePath,
 		return vc.Process{}, err
 	}
 
+	// Add the ephemeral device if ephemeral volume
+	// has to be attached to the container
```

Comment:
Maybe we could be a bit more verbose here because we could think that adding this ephemeral storage might fail if does not exist yet, but the agent actually takes care of creating the volume inside the VM if it does not exist yet, and does nothing if it already exists.